src/elements/featured-videos.html

```diff
@@ -11,54 +11,46 @@
 
 <dom-module id="featured-videos">
   <template>
-    <style is="custom-style" include="shared-styles flex flex-alignment positioning scale-animations"></style>
+    <style is="custom-style" include="shared-styles flex flex-alignment positioning"></style>
 
     <style>
       :host {
         display: block;
-        --video-item-height: 256px;
+        --video-item-height: 200px;
       }
 
-      .video-list {
+      .videos-wrapper {
         position: relative;
         overflow: hidden;
       }
 
+      .video-list {
+        flex-wrap: nowrap;
+        overflow-x: auto;
+        -webkit-overflow-scrolling: touch;
+      }
+
+      .video-list::-webkit-scrollbar {
```

Comment:
Does not work on Firefox
![image](https://user-images.githubusercontent.com/2954281/37876849-761908c0-305b-11e8-9784-a0d1042ac67d.png)

Reply:
fixed, thx for bringing it up